src/game/Strategic/Quest_Debug_System.cc

```diff
@@ -2734,7 +2734,7 @@ static void EnableQDSButtons(void)
 
 	if( gfUseLocalNPCs )
 	{
-		bool const enable = IsMercInTheSector(gubCurrentNpcInSector[gNpcListBox.sCurSelectedItem]) != -1;
```

Comment:
You could also just remove the extraneous extra scope at the start of the function and use the result here for brevity.

Reply:
I don't want to. =~
I avoid restructuring in coverity PRs to make sure the original problem is easy to see.